kubernetes/data_source_kubernetes_service_test.go

```diff
@@ -75,6 +83,8 @@ func testAccKubernetesDataSourceServiceConfig_basic(name string) string {
     }
   }
   spec {
+	ip_families = ["IPv4"]
+	ip_family_policy = "SingleStack"
```

Comment:
These two values are default and I put them here to make the test looks more explicit.